tests/test-sources/plugins/lsp/efmls-configs.nix

```diff
@@ -1,8 +1,50 @@
-{
+{efmls-options, ...}: {
   empty = {
     plugins.efmls-configs.enable = true;
   };
 
+  all = {
+    plugins.efmls-configs = let
+      options = efmls-options.options.plugins.efmls-configs;
+      toolOptions = (builtins.head options.setup.type.getSubModules).options;
+
+      brokenTools = [
+        #Broken as of 16 of November 2023
+        "phpstan"
+      ];
+
+      unpackaged = [
+        "blade_formatter"
+        "cspell"
+        "dartanalyzer"
+        "debride"
+        "fecs"
+        "fixjson"
+        "forge_fmt"
+        "gersemi"
+        "js_standard"
+        "pint"
+        "prettier_eslint"
+        "prettier_standard"
+        "redpen"
+        "reek"
+        "rome"
+        "slim_lint"
+        "solhint"
+        "sorbet"
+        "xo"
+      ];
+
+      toolsFromOptions = opt:
+        builtins.filter (t: !builtins.elem t (brokenTools ++ unpackaged))
+        opt.type.nestedTypes.right.nestedTypes.elemType.nestedTypes.left.functor.payload;
+    in {
+      enable = true;
+
+      setup = builtins.mapAttrs (_: builtins.mapAttrs (_: toolsFromOptions)) toolOptions;
```

Comment:
I wouldn't mind having either (or both) a little clearer implementation (yet more verbose) or some comments to explain what is going on.
(Especially for the `setup` line).

Reply:
Done! I agree that it's better, finding the correct invocation was a lot of `pkgs.lib.debug.traceSeqN`